src/components/SettingsPanel.vue

```diff
@@ -91,6 +91,28 @@
           </option>
         </select>
       </div>
+      <div>
+        <label
+          class="settings-panel--text"
+          @mouseover="help('hostKeyboardLayout')"
+          :title="$t('settingsPanel.hostKeyboardLayout.title')"
+          >{{ $t('settingsPanel.hostKeyboardLayout.title') }}</label
+        >
+      </div>
+      <div>
+        <select
+          id="setting-panel-host-keyboard-layout"
+          v-model="hostKeyboardLayout"
+        >
+          <option
+            v-for="osLayout in hostKeyboardLayouts"
+            :key="osLayout"
+            :value="osLayout"
+          >
```

Comment:
It would have been more readable in my opinion to keep it the options of a tag in a single line but I can't commit without calling the linter ¯\_ (ツ)_/¯